docs/source/tutorials/getting-started/intro.rst

```diff
@@ -5,14 +5,15 @@ Getting Started
 ###############
 
 .. toctree::
-   :hidden:
+   :titlesonly:
```

Comment:
Just wondering if you checked this against the `docs.daml.com` repo somehow. The final index will be created there

Reply:
I have reverted this change. I wanted to try out something by aligning this setting with the previous main sections, but it did not do want I was looking for.